content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -175,6 +175,21 @@ In case the discovery file does not contain credentials, the TLS discovery token
   you must keep it secret and transfer it over a secure channel. This might be possible with your
   cloud provider or provisioning tool.
 
+{{< note >}}
+To allow `kubeadm join` to use predefined kubelet credentials and skip client TLS bootstrap
+and CSR approval for a new node:
+1. From a working control plane node in the cluster that has `/etc/kubernetes/pki/ca.key`
+execute `kubeadm kubeconfig user --org system:nodes --client-name system:node:$NODE > kubelet.conf`.
+  `$NODE` must be set to the name of the new node.
```

Comment:
The correct indentation:

```
1. something first line
   line continues ...
   more lines ...
```

This is to avoid being trapped into markdown indentation issues.
Always indent the same paragraph at the same level make the list easy to maintain.

Reply:
> This is to avoid being trapped into markdown indentation issues.
> Always indent the same paragraph at the same level make the list easy to maintain.

oK @tengqm, I will correct it.

Reply:
I meant "correcting the source on line 182 (by adding 3-spaces indentation), and line 185.

By the way, I don't think we want to call these paragraphs into a `{{note}}`.

Reply:
> I meant "correcting the source on line 182 (by adding 3-spaces indentation), and line 185.
Ok.

> By the way, I don't think we want to call these paragraphs into a {{note}}.

@tengqm, So should I remove this from the note and mention or append it with the content of [File or HTTPS-based discovery](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/#file-or-https-based-discovery) 

but @neolit123 wants it in the note section, as per mention in the issue.

@neolit123, your thoughts on this?

Reply:
my thinking was to have it as an optional setup in the join docs, so "note" is one way. alternatively it could be a sub section of the file / http section.

Reply:
how about a sibling section (H4) next to "File/HTTP based discovery"?
These new paragraphs are about using existing credentials rather than discovering them from a given source.

Reply:
Thanks, @tengqm and @neolit123 for your comments, I will update it accordingly.

Reply:
Hi @tengqm @neolit123 
I have addressed the comments.
PTAL!